cernopendata/jsonschemas/records/record-v1.0.0.json

```diff
@@ -64,54 +69,67 @@
       "type": "object"
     },
     "accelerator": {
+      "description": "The accelerator involved in the production of the data (e.g. CERN-LHC)",
       "type": "string"
     },
     "authors": {
       "items": {
         "properties": {
           "affiliation": {
+            "description": "The affiliation of the author",
             "type": "string"
           },
           "ccid": {
+            "description": "The author's CCID",
             "type": "string"
           },
           "inspireid": {
+            "description": "The author's inspire ID",
             "type": "string"
           },
           "name": {
+            "description": "The name of the author",
             "type": "string"
           }
         },
+        "required": ["name"],
```

Comment:
Getting validation errors on `required` parts:

```console
$ python ../data-curation/cod2-to-cod3/code/validate_records.py                                                                2 ↵
cms-validated-runs.json
Traceback (most recent call last):
  File "../data-curation/cod2-to-cod3/code/validate_records.py", line 15, in <module>
    jsonschema.validate(record, schema)
  File "/usr/lib/python3.7/site-packages/jsonschema/validators.py", line 540, in validate
    cls.check_schema(schema)
  File "/usr/lib/python3.7/site-packages/jsonschema/validators.py", line 83, in check_schema
    raise SchemaError.create_from(error)
jsonschema.exceptions.SchemaError: ['name'] is not of type 'boolean'

Failed validating 'type' in schema['properties']['properties']['additionalProperties']['properties']['required']:
    {'default': False, 'type': 'boolean'}

On instance['properties']['collaboration']['required']:
    ['name']
```

We are using JSON Schema version 3 in the file header, where the `required` property was only Boolean it seems https://tools.ietf.org/html/draft-zyp-json-schema-03#section-5.7

Perhaps we can upgrade to v4 everywhere:

```console
$ git grep draft
cernopendata/jsonschemas/records/data-policies-v1.0.0.json:  "$schema": "http://json-schema.org/draft-04/schema#",
cernopendata/jsonschemas/records/data-policies-v1.0.0.json:        "$schema": "http://json-schema.org/draft-04/schema#",
cernopendata/jsonschemas/records/datasets-v1.0.0.json:  "$schema": "http://json-schema.org/draft-03/schema#",
cernopendata/jsonschemas/records/docs-v1.0.0.json:  "$schema": "http://json-schema.org/draft-04/schema#",
cernopendata/jsonschemas/records/docs-v1.0.0.json:        "$schema": "http://json-schema.org/draft-04/schema#",
cernopendata/jsonschemas/records/glossary-term-v1.0.0.json:  "$schema": "http://json-schema.org/draft-04/schema#",
cernopendata/jsonschemas/records/record-v1.0.0.json:  "$schema": "http://json-schema.org/draft-03/schema#",
cernopendata/jsonschemas/records/software-v1.0.0.json:  "$schema": "http://json-schema.org/draft-03/schema#"
```

Reply:
Done.